include/net/nrf_cloud_pgps.rst

```diff
@@ -104,15 +105,19 @@ Requesting and processing P-GPS data
 P-GPS data can be requested from the cloud using one of the following methods:
 
 * Directly:
-
-  * By calling the function :c:func:`nrf_cloud_pgps_request_all` to request a full set of predictions.
-  * By passing a properly initialized :c:struct:`gps_pgps_request` structure to the :c:func:`nrf_cloud_pgps_request` function.
+  * If :option:`CONFIG_NRF_CLOUD_MQTT` is enabled:
+    * By calling the function :c:func:`nrf_cloud_pgps_request_all` to request a full set of predictions.
+    * By passing a properly initialized :c:struct:`gps_pgps_request` structure to the :c:func:`nrf_cloud_pgps_request` function.
+  * If :option:`CONFIG_NRF_CLOUD_REST` is enabled:
+    * By passing a properly initialized :c:struct:`nrf_cloud_rest_pgps_request` structure to the :c:func:`nrf_cloud_rest_pgps_data_get` function.
 
 * Indirectly:
-
-  * By calling :c:func:`nrf_cloud_pgps_init`, with no valid predictions present in flash, or with some or all of the predictions expired.
-  * By calling :c:func:`nrf_cloud_pgps_preemptive_updates`.
-  * By calling :c:func:`nrf_cloud_pgps_notify_prediction`.
+  * If :option:`CONFIG_NRF_CLOUD_MQTT` is enabled:
+    * By calling :c:func:`nrf_cloud_pgps_init`, with no valid predictions present in flash, or with some or all of the predictions expired.
```

Comment:
`nrf_cloud_pgps_init` needs to be called regardless of transport, though maybe support for this is incomplete until @jtguggedal 's PR is in place?

Reply:
yes, but this section is for:
`P-GPS data can be requested from the cloud using one of the following methods:`

Reply:
OK, fair enough.